src/events/onReactionRemovepdateStarboard.evt.ts

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (c) 2024 Inimi | DrHooBs | Incoverse
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <https://www.gnu.org/licenses/>.
+ */
+
+import { DrBotEvent, DrBotEventTypeSettings, DrBotEventTypes } from "@src/lib/base/DrBotEvent.js";
+import { Client, Embed, EmbedBuilder, Events, Guild, TextChannel } from "discord.js";
+
+// DrVem's Community Server <:MrSzanto:1277693330280550450>
+// Test Server <:MrSzanto:1277693330280550450>
+export default class ORRUS extends DrBotEvent {
+    
+    protected _type: DrBotEventTypes = "discordEvent";
+
+    protected _typeSettings: DrBotEventTypeSettings = {
+        listenerKey: Events.MessageReactionRemove,
+    }
+
+    protected _priority: number = 0;
+    
+
+    public async runEvent(reaction, user, client: Client) {
+        if (reaction.partial) {
+            try {
+                await reaction.fetch();
+                await user.fetch();
+            } catch (error) {
+                console.error("Something went wrong when fetching the message: ", error);
+                return;
+            }
+        }
+        let embed = new EmbedBuilder()
+            .setAuthor({
+                name: reaction.message.author.username,
+                iconURL: reaction.message.author.displayAvatarURL()
+            })
+            .setDescription(reaction.message.content)
+            .addFields({
+                name: "Source",
+                value: `[Jump to message](${reaction.message.url})`
+            })
+            .setFooter({
+                text: `${reaction.message.id} • DrBot's Starboard`
+            })
+
+
+        if (reaction.emoji.name == "MrSzanto") {
+            
+            let guild = reaction.message.guild as Guild;
+            let starboardChannel = guild.channels.cache.find(channel => channel.name === "starboard");
```

Comment:
This is not gonna work due to the fact that the channels in DrVem's Community Centre start with "``»・``", a better way would be to add a regex check, for example:
```js
/star(-){0,1}board/gi
```
which would trigger for channels such as: ``»・starboard``, ``starboard``, ``»・star-board``, and ``star-board``

```suggestion
            let starboardChannel = guild.channels.cache.find(channel => /star(-){0,1}board/gi.test(channel.name)); // Find a channel that has "starboard" in the name (varying formats)
```